### PR TITLE
Front end robot control

### PR DESCRIPTION
**Pull request by thomassimko**

### CODE REVIEW

File: frontEnd/src/js/models/LoginController.ts [hunk 1]
```diff
@@ -34,15 +39,42 @@ export class LoginController implements ILoginController {
         try {
             const key:string = await this.serverController.postRequest("api/login", usernamePasswordJson);
             if (!isNullOrUndefined(key)) {
+                const decodedKey = (jwtDecoder(key));
+                console.log(decodedKey);
                 this.cookies.set('QuickPick', key);
+                //this.getAccessToken();
+                //this.getRefreshToken();
+                //this.getIdToken();
```
Comment (mdavis60) [inline frontEnd/src/js/models/LoginController.ts]: remove this maybe? unless we are keeping it for next sprint

File: frontEnd/src/js/models/devices/DeviceConnection.ts [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+            //     secretAccessKey: AWS.config.credentials.secretAccessKey,
+            //     sessionToken: AWS.config.credentials.sessionToken,
+            //     endpointAddress: 'a3dam0y9od060l.iot.us-west-2.amazonaws.com',
+            //     region: 'us-west-2'
+            // });
+            // console.log(mqttClient);
+            //
+            // mqttClient.on('connect', () => {
+            //     console.log('connected to iot mqtt websocket');
+            // });
```
Comment (mdavis60) [inline frontEnd/src/js/models/devices/DeviceConnection.ts]: remove this too?

File: frontEnd/src/js/models/identification/IdentificationController.ts [hunk 3]
```diff
@@ -23,19 +23,19 @@ export class IdentificationController implements IIdentificationController {
...
 
-                shoulder2X: JSON.xCoordinates[i+2],
-                shoulder2Y: JSON.yCoordinates[i+2],
-            }
+                shoulder2X: JSON.xCoordinates[i + 2],
+                shoulder2Y: JSON.yCoordinates[i + 2],
+            };
             try {
                 const key:string = await this.server.postRequest("api/berryInformation", berry);
             } catch (e) {
```
Comment (mdavis60) [inline frontEnd/src/js/models/identification/IdentificationController.ts]: this is all going to be changed in another branch and may cause merge conflicts

File: frontEnd/src/js/react-components/devices/DeviceTable.tsx [hunk 4]
```diff
@@ -95,13 +95,9 @@ export class DeviceTable extends React.Component<IDeviceTableProps, IDeviceTable
...
      */
     render() {
-        // Map the devices stored in the state from JSON form to a Device so
-        // we can utilize the methods defined in that class
-
-        const data = this.mapDevicesToData();
         return <div>
             <ReactTable
-                data={data}
+                data={this.mapDevicesToData()}
```
Comment (johnnicholson) [inline frontEnd/src/js/react-components/devices/DeviceTable.tsx]: better

**Pull request merged**